packages/common/src/store/pages/trending/reducer.ts

```diff
@@ -57,17 +65,7 @@ const trendingAllTimeReducer = asLineup(
   trendingAllTime
 )
 
-const reducer = (state, action) => {
```

Comment:
gotcha so this is the part that is replaced with not-lazy version to ensure each lineup is ready to go when a repost action happens?

Reply:
yeyo